soc/st/stm32/Kconfig.defconfig

```diff
@@ -20,6 +20,21 @@ config CLOCK_CONTROL
 config CORTEX_M_SYSTICK
 	default n if STM32_LPTIM_TIMER
 
+# Reduce kernel stack sizes to fit MCUs with 8 KiB RAM or less
+config MAIN_STACK_SIZE
+	default 320 if SRAM_SIZE <= 2
+	default 512 if SRAM_SIZE <= 4
+	default 640 if SRAM_SIZE <= 8
+
+config IDLE_STACK_SIZE
+	default 100 if SRAM_SIZE <= 2
+	default 150 if SRAM_SIZE <= 4
```

Comment:
Despite this change is synced with the value previosly used in applicable SoCs, I find it strange the stack size is not a multiple of a 32bit word size.
This comment is outside the scope of this P-R, not blocking.